mica-rest/src/main/java/org/obiba/mica/AbstractGitPersistableResource.java

```diff
@@ -116,11 +119,10 @@ public Response getShareKey(@QueryParam("expire") String expire) {
   @Path("/_share")
   public Response getShareURL(@QueryParam("expire") String expire) {
     checkPermission("/draft/" + getService().getTypeName(), "EDIT");
-    return Response.ok().entity(String.format("https://portal.example.org/mica/%s/%s/draft/%s",
-        getService().getTypeName(), getId(), createShareKey(expire))).build();
+    return Response.ok().entity(String.format("%s/%s/%s/draft/%s",
```

Comment:
And you should also get the pattern from the application yml (hidden), something like:

${portal}/mica/${type}/${id}/draft/${key}